app/server.py

```diff
@@ -25,12 +25,8 @@ def index():
 
 def bad_request(err):
     message = {'Error': err[1], 'Status Code': err[0]}
-    if(err[2] == 'xml'):
-        response = dicttoxml(message)
-    else:
-        response = json.dumps(message)
-    resp = make_response(response, err[0])
-    return resp
+    response = dicttoxml(message) if err[2] == 'xml' else json.dumps(message)
+    return make_response(response, err[0])
 
```

Comment:
Does it make any difference.

Reply:
Perhaps run dis.dis() on the code and see.